### ISSUE TITLE
Safari: CC/subtitles broken

### ISSUE THREAD

**Issue by bhh1988** [google/shaka-player#991]
I am on Safari 10.1.2, and HEAD of master.

When I try to load a video with VTT subtitles declared in the manifest (using BaseURL, not segmented, but I doubt it matters), there is an error thrown on simple_text_displayer.js:194.

```
  if (shakaCue.textAlign == 'center' && vttCue.align != 'center') {
    // Workaround for a Chrome bug http://crbug.com/663797
    // Chrome does not support align = 'center'
    vttCue.position = 'auto';
    vttCue.align = 'middle';
  }
```

The error thrown says "The provided value is non-finite". I think it's similar to this issue here: https://github.com/mozilla/vtt.js/issues/364

And then subtitles aren't shown on Safari at all, though the video/audio will continue playing.

**Comment by joeyparrish** [comment]
We need to reproduce locally, file a bug on Apple if there isn't one yet, then work on a polyfill for Safari to fix it in JS.  Thanks for the report!

**Comment by joeyparrish** [comment]
The fix will be cherry-picked to v2.2.2.
